### DISCUSSION TITLE
submitting the output from colab

### DISCUSSION THREAD

**Discussion by olliethomas** [wearepal/acre-cascade#14] [Show and tell]
![image](https://user-images.githubusercontent.com/1835492/102102389-a6e90a80-3e23-11eb-91fc-4dee7d67107f.png)
Just in case anyone is under the mistaken impression that the colab notebook can return something _not_ terrible with the current settings 😂

**Comment by tmke8** [comment]
Is 0.0275 bad?

**Reply by olliethomas** [reply]
Yeah, the score is the [Jaccard Index](https://en.wikipedia.org/wiki/Jaccard_index), so 0=bad, 1=good
